### ISSUE TITLE
Multicraft-Portals Won't enable with Uskyblock

### ISSUE THREAD

**Issue by RhysRS** [rlf/uSkyBlock#855]
_Please paste the output from `/usb version` below
![2016-08-15_00 57 25](https://cloud.githubusercontent.com/assets/21028508/17652994/09790734-6284-11e6-81c3-702ad341f01c.png)

_What steps will reproduce the problem?_
1. downloading Uskyblock 2.7.2
2. downloading Multiverse-Portals
3. multiverse-portals disables
4. nether doesn't work

_If you have any log-files, please paste them to [pastebin.com](http://pastebin.com)_
* server-log: http://pastebin.com/ycYHMnwj

**Comment by dutchy1001** [comment]
**Multiverse-Portals** is a depended plugin as it work together with Multiverse-Core and Multiverse_NetherPortals.
I suggest to swap to Multiverse-portals instead of Multicraft-Portals.

**Comment by RhysRS** [comment]
Sorry, I meant Multiverse-Portals.

**Comment by rlf** [comment]
Thanks for using pastebin, and including a full server-log.

If you take some time actually reading that server-log, you will get some nice hints as to what is causing this.

First is line 11, where a plugin called `NCWarpPortal` fails to load.
2nd is line 371, where `Multiverse-Core' complains - indicating it's an old (non 1.9+ compliant) version.
3rd is line 409, where `WarpPortals` (how many portals plugins do you need?) also tells you it's not compliant with your current minecraft version.
4th is line 424, where `WarpSigns` .. sigh... you get the picture...

Actually, one of the few plugins that are NOT failing, and NOT causing an exception, is uSkyBlock.

Bottom line is, update your Multiverse-plugins to something compatible with your minecraft-server version, and uSkyBlock should start working again.

Rejected.
